src/signal.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2011-2016 Swift Navigation Inc.
+ * Contact: Jacob McNamee <jacob@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef SWIFTNAV_SIGNAL_H
+#define SWIFTNAV_SIGNAL_H
+
+#include <libswiftnav/common.h>
+#include <libswiftnav/signal.h>
+
+/* Platform-specific code support ==== */
```

Comment:
Yes, definitely. Either makefile or board.h. Just waiting for other changes to set this up.